pyexchange/erisx.py

```diff
@@ -223,30 +230,13 @@ def get_trades(self, pair: str, page_number: int = 8) -> List[Trade]:
         response = self._http_post("trades", {"account_id": self.account_id})
         return list(map(lambda trade: ErisxTrade.from_message(trade), response["trades"]))
 
-    # used to unsubscribe from the marketdata feed used in get_all_tradesx
-    def unsubscribe_marketdata(self, pair: str, request_id: str) -> bool:
-        assert(isinstance(request_id, str))
-
-        message = self.fix_marketdata.create_message(simplefix.MSGTYPE_MARKET_DATA_REQUEST)
-        
-        message.append_pair(262, request_id)
-        message.append_pair(263, 2)
-        message.append_pair(264, 0)
-        message.append_pair(266, 'Y')
-        message.append_pair(267, 2)
-        message.append_pair(269, 0)
-        message.append_pair(269, 1)
-        message.append_pair(146, 1)
-        message.append_pair(simplefix.TAG_SYMBOL, self._format_pair_string(pair))
-        
-        self.fix_marketdata.write(message)
-        response = self.fix_marketdata.wait_for_response('8')
-        return True if response else False
+    # TODO: Not currently available
+    def get_all_trades(self, pair, page_number) -> List[Trade]:
+       pass
 
-    # TODO: finish implementing
-    def get_all_trades(self, pair: str, page_number: int = 1) -> List[Trade]:
-        assert(isinstance(pair, str))
-        assert(isinstance(page_number, int))
+    def get_orderbook(self, pair: str, page_number: int = 1) -> List[Trade]:
```

Comment:
I don't believe we should be creating `Trade` objects for this.  Other exchange implementations seem to return raw data from the exchange API.  Please see other comment in `ErisxFix.parse_order_book`.

Reply:
Updated typing accordingly